src/core/persistence/Codend.Persistence/Repositories/SprintProjectTaskRepository.cs

```diff
@@ -0,0 +1,26 @@
+using Codend.Domain.Entities;
+using Codend.Domain.Repositories;
+
+namespace Codend.Persistence.Repositories;
+
+public class SprintProjectTaskRepository
+    : GenericRepository<SprintProjectTaskId, Guid, SprintProjectTask>,
+        ISprintProjectTaskRepository
+{
+    public SprintProjectTaskRepository(CodendApplicationDbContext context) : base(context)
+    {
+    }
+
+    public IEnumerable<SprintProjectTask> GetRangeBySprintIdAndProjectTaskIds(
```

Comment:
Casting IQueryable to IEnumerable will result in lazy fetching data synchronously. We don't want that. Call ToListAsync() and return without await to let user decide when to await data and not to block processor.